pingtop.py

```diff
@@ -344,6 +345,10 @@ def multi_ping(host, packetsize, logto, log_level):
     hosts = {h: {} for h in host}
     logger.info(f"Hosts: {hosts}")
     worker_num = len(hosts)
+    if worker_num == 0:
+        logger.error("Hosts is not defined.")
+        logger.error("Usage: pingtop host1 [host2] [host3] ...")
```

Comment:
need sudo
"Usage: sudo pingtop host1 [host2] [host3] ..."

Reply:
It would be better if we raise `click.BadParameter` here, it will give useful information and set property exit value here.

Log if for debug and when set `logto` option user will not see log in stdout.